oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -124,30 +166,37 @@ volunteers are needed, we will put out a call to join. Volunteers will need to a
 and will then be evaluated by the existing working group members. We hope to have six to seven members of the working
 group at a time.
 
-Security work will be a mix of private and public tickets
----------------------------------------------------------
-Proactive work for security improvements will be made public, while vulnerability reports will be handled privately.
-We propose using Github Security Advisories to handle the triage of vulnerability reports to the owners of
```

Comment:
@feanil I think for GitHub security advisories to work for us, we will need to grant maintainers repository ownership / GitHub repository admin permissions on this repo. Is this in alignment with tCRIL's plans for maintainership?

Reply:
(What I am reading: https://docs.github.com/en/code-security/security-advisories/repository-security-advisories/permission-levels-for-repository-security-advisories)

Reply:
@pshiu I think there are a couple of ways to solve this but at the very least I expect the security working group will need admin access to all the open source repos to do their job.  I think we're okay with this.

For the larger question of maintainers and admin,  I'm gonna think about it more but it's come up in a few other places as well so I think we'll probably add an ADR to the maintainers OEP soon.

Reply:
Always happy with more permissions, but there is a beta Security Manager role GitHub offers too that we might be able to tinker around with. [[docs](https://docs.github.com/en/organizations/managing-peoples-access-to-your-organization-with-roles/managing-security-managers-in-your-organization)]